src/components/MoneyRequestConfirmationList.js

```diff
@@ -755,6 +755,7 @@ function MoneyRequestConfirmationList(props) {
                                     ? translate('common.error.enterMerchant')
                                     : ''
                             }
+                            rightLabel={translate('common.required')}
```

Comment:
Don't we need the `PolicyUtils.isPaidGroupPolicy` check here?

Reply:
@cead22  My understanding is that this `Merchant` `MenuItem` is only shown whenever `isPolicyExpenseChat`, where in the case of P2P its a little further down.

This `isMerchantRequired` ultimately is reliant up on the report chat type being `policyExpenseChat` [here is the code link](https://github.com/Expensify/App/blob/784aa27a848867d85ff5f927d4267e2b84af747f/src/libs/ReportUtils.ts#L777) (the `isPaidGroupPolicy` check is a little further down. Are we looking expense chat reports that are exclusively from Control or Collect policies?

Reply:
Ah, yes, I see that `isMerchantRequired` for the case of `src/components/MoneyTemporaryForRefactorRequestConfirmationList.js`. For this one we have `shouldShowMerchant` and I don't see it relying on `policyExpenseChat`, but if it's showing for workspace requests and not showing for p2p requests, we're good

Reply:
Yeah the dependency of that check goes `isMerchantRequired` > `isPolicyExpenseCheck` > `ReportUtils.isPolicyExpenseChat` (in the parent component). This file isn't up to date it looks like but the temporary other file is!

Reply:
Nice 👌 

> This file isn't up to date it looks like but the temporary other file is!

Do we need to update this file, or what do you mean here?

Reply:
I've updated this file, the original `MoneyRequestConfirmationList` file, whenever I've made a change in the temporary `MoneyTemporaryForRefactorRequestConfirmationList` file, but it looks like the temporary file is now being treated as the source of truth for this component and now the original file is out of date. We don't need to make any more changes!